### PR TITLE
13946-libh3-is-always-rebuilt

### PR DESCRIPTION
**Pull request by frolui**

### CODE REVIEW

File: openstreetmap-install.sh [hunk 1]
```diff
@@ -36,10 +36,16 @@ sudo apt install -y postgresql-server-dev-$pg_version
 # Installing pgxnclient
 sudo apt install -y pgxnclient
 ## pgxn has to call pg_config, add PG pinaries to PATH
-sudo echo 'export PATH=$PATH:/usr/local/pgsql/bin' >> /root/.bashrc
+## sudo echo 'export PATH=$PATH:/usr/local/pgsql/bin' >> /root/.bashrc
 
 # Installing H3
-sudo pgxn install h3
+if [ -d "/usr/lib/postgresql/$pg_version/lib/bitcode/h3/" ] && [ -e "/usr/lib/postgresql/$pg_version/lib/bitcode/h3.index.bc" ] && [ -e "/usr/lib/postgresql/$pg_version/lib/h3.so" ]
```
Comment (lgudyma-kontur) [inline openstreetmap-install.sh:42]: have can we recreate it if we needed at some point of time?

Comment (kalenikaliaksandr) [inline openstreetmap-install.sh:42]: I think checking output of `psql -c "select h3_get_extension_version()"` will be more clear and reliable way.

**Review by kalenikaliaksandr — CHANGES_REQUESTED** [review]

File: openstreetmap-install.sh [hunk 2]
```diff
@@ -36,10 +36,16 @@ sudo apt install -y postgresql-server-dev-$pg_version
 # Installing pgxnclient
 sudo apt install -y pgxnclient
 ## pgxn has to call pg_config, add PG pinaries to PATH
-sudo echo 'export PATH=$PATH:/usr/local/pgsql/bin' >> /root/.bashrc
+## sudo echo 'export PATH=$PATH:/usr/local/pgsql/bin' >> /root/.bashrc
```
Comment (kalenikaliaksandr) [inline openstreetmap-install.sh:39]: if this code should not be executed then you need to remove it. there is git history if you will want to find this code in the future.

File: openstreetmap-install.sh [hunk 3]
```diff
@@ -49,7 +55,7 @@ sudo apt install -y postgresql-common
 set_system_locale
 
 ## Create cluster and systemd service
-sudo pg_createcluster $pg_version main 
+#sudo pg_createcluster $pg_version main
```
Comment (kalenikaliaksandr) [inline openstreetmap-install.sh:58]: please no commented code

**Review by kalenikaliaksandr — CHANGES_REQUESTED** [review]

File: openstreetmap-install.sh [hunk 4]
```diff
@@ -36,10 +36,16 @@ sudo apt install -y postgresql-server-dev-$pg_version
 # Installing pgxnclient
 sudo apt install -y pgxnclient
 ## pgxn has to call pg_config, add PG pinaries to PATH
```
[the remark this answers is not in the document]
Comment (frolui) [reply openstreetmap-install.sh]: fixed

File: openstreetmap-install.sh (hunk 1 above)
Comment (frolui) [reply openstreetmap-install.sh]: moved to separate installation makefile

Comment (frolui) [reply openstreetmap-install.sh]: rewrited as a target in installation makefile

File: openstreetmap-install.sh (hunk 2 above)
Comment (frolui) [reply openstreetmap-install.sh]: removed

**Review by kalenikaliaksandr — APPROVED** [review]

**Pull request merged**